tests/installation/partitioning_raid.pm

```diff
@@ -282,7 +282,12 @@ sub run {
             send_key 'right';
             send_key_until_needlematch "partitioning_raid-disk_vda-selected", "down";
             # edit first partition
-            send_key 'alt-e';
+            if (is_storage_ng) {
+                send_key 'alt-i';
+            }
+            else {
+                send_key 'alt-e';
+            }
```

Comment:
Would you be ok to change that to `send_key is_storage_ng ? 'alt-i' : 'alt-e';`?